spec/models/user_spec.rb

```diff
@@ -1,5 +1,66 @@
 require 'rails_helper'
 
+
 RSpec.describe User, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  user = User.where(email: 'test@example.com').first_or_create(password: '12345678', password_confirmation: '12345678')
```

Comment:
These tests look like the same tests in Destinations spec.  User model usually does not need validations or testing when using Devise as Devise has built-in validations already.